SalesforceSDK/Core/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="CommonServiceLocator" version="1.3.0" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="MvvmLightLibs" version="5.1.1" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="portable45-net45+win8+wpa81" />
```

Comment:
Yea, that's the point of this change... Nuget is telling these packages that it doesn't explicitly need support for WP8.1 when it pulls down these packages.  Some nuget packages have different binaries for different PCL profiles... this change is to bring SDK PCL profile in line with S1 PCL profile that we're using so we make sure same nuget packages are being used.  If you want to keep supporting WP8.1 then this change should not be made.  Note that Win10 is still supported with this change (actually a little bit better supported because removing the WP8.1 target opens up the API subset that this PCL profile uses a bit).